src/components/typography/typography.css

```diff
@@ -0,0 +1,33 @@
+h1,
+h2,
+h3,
+h5,
+h6 {
+  font-family: var(--ff-primary);
+}
```

Comment:
This is unnecessary here as it will be inherited by the `body` where you set the font already.